docker/conf/docker-compose-mac.yml.dist

```diff
@@ -43,13 +43,36 @@ services:
         ports:
             - "3000:3000"
 
+    # This service is used exclusively for running Cypress tests
```

Comment:
Do we really need this right above `storefront-cypress`? :)

Reply:
Probably not... I think the name is descriptive enough... I removed it.